packages/@tinacms/api-git/src/utils/paths.ts

```diff
@@ -16,21 +16,17 @@ limitations under the License.
 
 */
 
-import * as fs from 'fs'
 import * as path from 'path'
-import { openRepo } from './open-repo'
-
-export interface ShowConfig {
-  pathRoot: string
-  fileRelativePath: string
-}
-
-export async function show({ pathRoot, fileRelativePath }: ShowConfig) {
-  const repo = openRepo(pathRoot)
-
-  try {
-    return await repo.show([`HEAD:${fileRelativePath}`])
-  } catch (e) {
-    return fs.readFileSync(path.join(pathRoot, fileRelativePath), 'utf8')
-  }
+import * as os from 'os'
+
+export function checkFilePathIsInParent(filepath: string, parent: string) {
+  const fullpath = path.resolve(filepath)
+  const repopath = (() => {
```

Comment:
Instead of putting this in an immediately invoked function I think you could just do:

```ts
    let repopat: string
    if (os.type() === 'Windows_NT') {
      repopath = path.resolve(parent).replace(/\\+$/, '') + '\\'
    } else {
      repopath = path.resolve(parent).replace(/\/+$/, '') + '/'
    }
```